tools/active_vision/find_respawn_loc.py

```diff
@@ -0,0 +1,197 @@
+# get candidate image_ids, spawn location, target xyz
+from droidlet.perception.robot.active_vision.candidate_selection import SampleGoodCandidates
+from droidlet.lowlevel.robot_coordinate_utils import xyz_pyrobot_to_canonical_coords
+from droidlet.perception.robot.handlers import convert_depth_to_pcd, compute_uvone
+from common_utils import is_annot_validfn_class, is_annot_validfn_inst, log_time
+from typing import List
+
+import os 
+import glob
+import json
+import matplotlib.pyplot as plt 
+import cv2
+import numpy as np
+import argparse
+import submitit
+import shutil
+
```

Comment:
at a high level, it would help me understand this script if the logic for loading files was separated from the processing in the functions.   it seems like almost every function here has its own loading boilerplate